README.md

```diff
@@ -1,2 +1,55 @@
-# sdl_app_library_guides
-DocDown Guide Content for smartdevicelink.com
+# SDL App Library Guide Content
+iOS and Java Suite (Android, JavaEE, and JavaSE) guide content for the [SDL Developer Portal](https://smartdevicelink.com/en/guides/iOS/getting-started/installation/).
+
+## Overview
+Documents in this guide are created in a custom SDL markdown format. All guides get created as directories inside this repository. Each directory gets a link in the left-hand sidebar on [smartdevicelink.com](https://smartdevicelink.com). Guides do not currently support multiple pages or sub-directories. Guides specific to one platform or with content specific to one platform will use custom tags to denote this content.
+
+Learn how to use SDL markdown (aka DocDown) at the [spec page](https://github.com/smartdevicelink/sdl_markdown_spec).
+
+## Custom SDL Markdown Tags
+Basic markdown tags will [as expected](https://github.com/smartdevicelink/sdl_markdown_spec#basic-markdown-syntax) including headings, bold, italics, and tables. However, there are [custom extensions](https://github.com/smartdevicelink/sdl_markdown_spec#basic-markdown-syntax) for inline notes and relative links to assets.
+
+### Specifying Platforms
+This repository has additional customizations to permit content to be included or excluded from one of the several platforms of documentation that this repository builds. This can be done inline within a guide in order to, for example, display one code snippet for iOS and another for Java-family platforms. This can also be done within the configuration file in order to allow a guide such as deploying a Java Web app to AWS to only build for the JavaEE platform.
```

Comment:
```suggestion
This repository has additional markdown customizations to permit content to be included or excluded from a platform. This can be done inline within a guide in order to, for example, display one code snippet for iOS and another for Java-family platforms. This can also be done within the configuration file in order to allow a guide, such as a guide demoing how to deploy a Java Web app to AWS, to only build for the JavaEE platform.
```